app/helpers/earthworks_blacklight_helper.rb

```diff
@@ -7,4 +7,10 @@ def show_content_classes
   def show_sidebar_classes
     'page-sidebar col-lg-2'
   end
+
+  def render_details_links(args)
+    args[:document].identifiers.map do |identifier|
+      link_to identifier, identifier
+    end.join(', ').html_safe
```

Comment:
rubocop doesn't like `.html_safe` -- is there a better way of doing this via some blacklight magic (i.e. `add_show_field` can render links)